.idea/jarRepositories.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
These additions aren't required. Please remove these from this PR

Reply:
@Priyansh-Kedia can you make separate pull request addressing this problem, as I really don't think there's need for these additions. For this PR, can you manually remove these additions?

Reply:
Removing the idea files removes multiple files and thus I dont think it should be done in the same PR.
I can raise another issue for the same.

Reply:
@Priyansh-Kedia removing an additional file from commit won't affect anything in my opinion. Also, adding `.idea` files will make project IDE specific which is not advised for any open source project

Reply:
Yes, that is right
I thought that this wasnt removed beforehand
I will update the PR soon
Thanks